lib/Test/Nginx/Socket.pm

```diff
@@ -3991,6 +3998,10 @@ One can enable HTTP/2 mode for an individual test block by specifying the L<http
 
     --- http2
 
+Enable SNI mode for an individual test block by specifying the L<sni> section, as in
+
+    --- sni
```

Comment:
I'm afraid `--- tls` is a better name than `--- sni` since we are not specifying a concrete SNI name here anyway.

Reply:
Thanks for your review and comments. I'll change it.